packages/strapi-design-system/src/Popover/Popover.js

```diff
@@ -122,6 +133,10 @@ const popoverProps = {
    * The callback invoked after a scroll to the bottom of the popover content.
    */
   onReachEnd: PropTypes.func,
+  /**
+   * The popover position
+   */
+  placement: PropTypes.string,
```

Comment:
I think this prop can be better defined, they're specific options available:

```ts
type Placement =
  | 'top'
  | 'top-start'
  | 'top-end'
  | 'right'
  | 'right-start'
  | 'right-end'
  | 'bottom'
  | 'bottom-start'
  | 'bottom-end'
  | 'left'
  | 'left-start'
  | 'left-end';
```

Reply:
Awesome thank you, I was looking for this 😄